docs/notebooks/plot_comparisons.ipynb

```diff
@@ -0,0 +1,1118 @@
+{
```

Comment:
<p>I'm having a look at 14.4 of ROS and I see they call it predictive comparison when it's divided by u^{hi} - u^{lo}. We could double-check this in our call.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/695/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yes, I also noticed that. I **do not** divide by the _u_ values. In their examples (pgs. 251-252), they also do not divide through _u_ but rather compute average predictive differences. 

I think we should keep the computation as average predictive differences in this first PR because: (1) it is easier to interpret, (2) the computation of average predictive comparisons when there are more than one contrast level being compared becomes more tricky because of the need to keep track of the indexing, and (3) the skilled and knowledge authors never provided a demo of average predictive comparisons (maybe this last point is too naive thinking).

Reply:
Fixed